hamamatsu.py

```diff
@@ -363,7 +374,7 @@ def init(self):
             ),
             dtype=np.uint16
         )
-        self.is_initialized = True
+        self.isInitialized = True
```

Comment:
both `self.is_initialized` and `self.isInitialized` exist in this branch and I assume they are meant to be the same variable. Convert `isInitialized` to `is_initialized`.

Reply:
see my prior comment.